playbooks/byo/openshift-prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: generate prometheus deployment
+  hosts: masters
```

Comment:
the tasks only need to run against a single master node (not all of them)

i'm seeing failures of this kind (presumably) because of this:

```
TASK [openshift_prometheus : Set alert and prometheus secrets] *****************
Thursday 27 July 2017  13:45:16 +0000 (0:13:20.345)       0:17:53.459 ********* 
changed: [master1] => (item=prometheus)
failed: [master2] (item=prometheus) => {"failed": true, "item": "prometheus", "msg": {"cmd": "/usr/bin/oc secrets new prometheus-proxy session_secret=/
tmp/session_secret-O7JqOq -n prometheus", "results": {}, "returncode": 1, "stderr": "Error from server (AlreadyExists): secrets \"prometheus-proxy\" already 
exists\n", "stdout": ""}}
failed: [master3] (item=prometheus) => {"failed": true, "item": "prometheus", "msg": {"cmd": "/usr/bin/oc secrets new prometheus-proxy session_secret=/t
mp/session_secret-zx9yIP -n prometheus", "results": {}, "returncode": 1, "stderr": "Error from server (AlreadyExists): secrets \"prometheus-proxy\" already e
xists\n", "stdout": ""}}
changed: [master2] => (item=alerts)
failed: [master1] (item=alerts) => {"failed": true, "item": "alerts", "msg": {"cmd": "/usr/bin/oc secrets new alerts-proxy session_secret=/tmp/session_s
ecret-mOz9Ct -n prometheus", "results": {}, "returncode": 1, "stderr": "Error from server (AlreadyExists): secrets \"alerts-proxy\" already exists\n", "stdou
t": ""}}
changed: [master3] => (item=alerts)
```